files/default/capture-agent-manager-logrotate.conf

```diff
@@ -1,8 +0,0 @@
-/home/capture_agent_manager/logs/*.log {
-	daily
-	missingok
-	rotate 7
-	compress
-	notifempty
-	copytruncate
-}
```

Comment:
Removing as I think we should just log to syslog (which requires changes to cadash `logging.yml`)